Run ./autogen.sh instead of autoconf.

IslandCow opened the pull request:
Fixes build for ghostscript.

jonathanmetzman commented:
For our records, the ghostscript problem is what is causing all of the AFL exceptions.

jonathanmetzman commented on projects/ghostscript/build.sh:
```diff
@@ -52,6 +52,7 @@ make -j$(nproc) libgs
 $CXX $CXXFLAGS $CUPS_LDFLAGS -std=c++11 -I. \
     fuzz/gstoraster_fuzzer.cc \
     -o "$OUT/gstoraster_fuzzer" \
+    -Wl,-rpath=. \
```
Does this work if I do this:
```
cd /some/path/not/containing/the/fuzzer
/path/to/the/fuzzer
```
?

jonathanmetzman replied:
I'm not a linking expert, but I think $ORIGIN might be what we need.

IslandCow replied:
No, I'm pretty sure that won't work.  I've changed it to $ORIGIN and I think it's correct now.

IslandCow commented:
PTAL.  I've updated the RPATH.  Tested by building in the container, moved contents of /out to /testing.  From /out, ran /testing/gstoraster_fuzzer.  It seems to work.

jonathanmetzman commented:
I think this worked.
